packages/shared/src/components/cards/AcquisitionFormCard.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactElement, useState } from 'react';
+import { useMutation } from '@tanstack/react-query';
+import { useRouter } from 'next/router';
+import { Card } from './Card';
+import { Radio } from '../fields/Radio';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { AcquisitionChannel, updateUserAcquisition } from '../../graphql/users';
+import { MiniCloseIcon } from '../icons';
+import { OnboardingTitleGradient } from '../onboarding/common';
+import { removeQueryParam } from '../../lib/links';
+
+const options = [
+  {
+    label: 'Referred by a friend or colleague',
+    value: AcquisitionChannel.Friend,
+  },
+  { label: 'Social media', value: AcquisitionChannel.SocialMedia },
+  { label: 'Search engine', value: AcquisitionChannel.SearchEngine },
+  { label: 'Blog, forum, or discussion', value: AcquisitionChannel.Blog },
+  {
+    label: 'Browser extension store',
+    value: AcquisitionChannel.ExtensionStore,
+  },
+  {
+    label: 'Advertisement or sponsorship',
+    value: AcquisitionChannel.Advertisement,
+  },
+  { label: 'Other', value: AcquisitionChannel.Other },
+];
+
+export const acquisitionKey = 'ua';
+
+export function AcquisitionFormCard(): ReactElement {
+  const [isDismissed, setIsDismissed] = useState(false);
+  const [value, setValue] = useState<AcquisitionChannel>();
+  const router = useRouter();
+  const onRemoveQueryParams = () => {
+    const removed = removeQueryParam(window.location.href, acquisitionKey);
+    router.replace(removed);
+  };
+  const onDismiss = () => {
+    setIsDismissed(true);
+    onRemoveQueryParams();
+  };
+
+  const { mutateAsync, isLoading } = useMutation(updateUserAcquisition, {
+    onSuccess: onRemoveQueryParams,
+  });
+
+  if (isDismissed) {
+    return null;
+  }
+
+  return (
+    <Card data-testid="acquisitionFormCard" className="p-4">
+      <OnboardingTitleGradient className="flex w-full flex-row items-center whitespace-nowrap pb-1 typo-body">
```

Comment:
Maybe rename this to be more generic?

Reply:
Which one? The `OnboardingTitleGradient `?

Reply:
Yes :D maybe just TitleGradient would do?

Reply:
The `OnboardingTitleGradient` was already named like this before. Renaming is tangential to this PR. If we want to do it, I'd rather do it in a followup, since it's not related to changes here.

Reply:
Happy with that :)
Just want to avoid not logging it somewhere 🙏

Reply:
Added to our backlog: https://dailydotdev.atlassian.net/browse/MI-190